Decode/PerfItemValue.cs

```diff
@@ -546,6 +546,7 @@ public ReadOnlySpan<byte> GetStringBytes(out Text.Encoding encoding)
                         case EventHeaderFieldEncoding.Value8:
                         case EventHeaderFieldEncoding.ZStringChar8:
                         case EventHeaderFieldEncoding.StringLength16Char8:
+                        case EventHeaderFieldEncoding.BinaryLength16Char8:
```

Comment:
Note: New encoding. Exactly the same as `StringLength16Char8` except for its default format, which is `HexBytes` instead of `StringUtf`.